models/staging/stg_fivetran_platform__incremental_mar.sql

```diff
@@ -17,7 +28,7 @@ fields as (
         updated_at,
         _fivetran_synced,
         incremental_rows
-    from base
+    from fields
 )
 
 select * 
```

Comment:
This query is still pulling from the result of the `fill_staging_columns` macro. I'm guessing you will want to update this part to:

```suggestion
select * 
from final
```

Reply:
Great catch, it seems this wasn't an issue because the fill_staging_columns macro has the connector_name regardless and therefore wasn't erroring out. Thanks for catching this and just updated.